README.md

```diff
@@ -228,6 +228,7 @@ You can modify the options for the plugin by passing in JSON upon instantiation.
 | cursorChar    | (string) The character used for the cursor. HTML works too! | pipe |
 | breakLines    | (boolean) Choose whether you want multiple strings to be printed on top of each other (`breakLines: true`), or if you want each string to be deleted and replaced by the next one (`breakLines: false`).  | true  |
 | nextStringDelay    | (number in milliseconds or array) The amount of time (milliseconds) between typing the next string when multiple strings are defined. You may either pass a number in milliseconds, or an array of values. The first value will be used as the delay before a new string starts, and the second value will be used as the delay after a string ends. For example, passing `[1000, 2000]` will tell TypeIt to pause 1000ms before typing a new string, and wait 2000ms after a string has just completed. | 750 |
+| autoStart    | (boolean) Defines if the instance start typing automatically. Turn this option to `false` if you want to create chain strings with the `type()`, `delete()` or `pause()` methods.  | true  |
```

Comment:
I don't think the "Turn this option `false`" sentence is needed -- `autoStart` should work the same regardless of how you're queuing up strings. Is this not how it works for you?